go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/mattermost/mattermost-server/v5
 go 1.12
 
 require (
-	github.com/Masterminds/squirrel v1.1.0
+	github.com/Masterminds/squirrel v1.2.0
```

Comment:
Any reason these minor versions are getting updated ? I would think they are unrelated.

Reply:
Thanks @agnivade , your feedback is always welcome.   Not sure how those versions got bumped - I accepted incoming changes for all my merges on go.mod then added the couple of new imports I needed.  I revert vendor again and re-apply my imports.

Reply:
Fixed via https://github.com/mattermost/mattermost-server/pull/13967/commits/c97286d8d199d6a65f743cf23395c35083bd0402 and https://github.com/mattermost/mattermost-server/pull/13967/commits/bc82508f28c36af4450b1e08455129a1f5ac9797   Thanks!